### DISCUSSION TITLE
Macbook display is 100% when waking from sleep, external monitor maintains last settings

### DISCUSSION THREAD

**Discussion by jaredbm** [MonitorControl/MonitorControl#1286] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Whenever I wake the macbook from sleep, the external monitor is the same as the way I left it but the macbook display is always set to 100%. Only when running this app

### Steps to reproduce

1. set the macbook display to 80%
2. let it got to sleep
3. wake up
4. you'll see it's 100%

### Expected behavior

The built in display is fully bright

### Environment Information (please complete the following information)

```markdown
- macOS version: Monterey
- Mac model:2019 Macbook pro 15"
- MonitorControl version: 4.0.2
- Monitor(s): built in Macbook display / 34" LG ultrawide
- Apple Silicon/M1 (yes or no): no
```

**Comment by waydabber** [comment]
Hi there - does this happen when you turn down the brightness to 80% via the macOS built in controls as well?

**Comment by jaredbm** [comment]
Hello, it seems to happen if the keyboard is used to turn it down. That is consistent. It sometimes (but rarely) happens if you use the built in controls and then wake from sleep.

**Comment by waydabber** [comment]
It seems like macOS does not register the changes made by (undocumented) API calls on this particular model. This does not seem to happen on an M1 Mac though.

**Comment by cambler** [comment]
I have the opposite issue, using an M1 with it's internal display as well as an external monitor. When waking from sleep, the MacBook display seems to retain it's brightness however the external display is at full blast, even though the slider in the app is at the position I set it at originally?

Really appreciate this app though. Fantastic work.

**Comment by waydabber** [comment]
You might have to change this setting to the highlighted one:

<img width="719" alt="Screen Shot 2022-05-18 at 22 56 09" src="https://user-images.githubusercontent.com/37590873/169154282-c1a40614-6398-4d92-bcad-31aac7d9ebc6.png">
 
Some displays do reset their brightness after sleep as they forget the values set via a DDC command. This is probably a firmware issue.

**Comment by jaredbm** [comment]
@waydabber The issue I have happens to the built in retina display on my Macbook, I don't know why that would be a firmware issue. And I do have that setting selected.

**Comment by waydabber** [comment]
sorry, I was responding to @cambler. His issue should be a separate one indeed but he hijacked this one. :)

**Comment by cambler** [comment]
Thanks so much @waydabber I'll try that out. Also apologies for the hijack, was actually trying to not cloud up the threads and thought this may be related.
